packages/ui/src/components/Tabs/STab.vue

```diff
@@ -0,0 +1,162 @@
+<script lang="ts">
+export default defineComponent({
+  name: 'STab',
+})
+</script>
+
+<script setup lang="ts">
+import { useTabsStateApi, TabsPanelApi } from './api'
+
+const props = withDefaults(
+  defineProps<{
+    disabled?: boolean
+    name: string
+  }>(),
+  {
+    disabled: false,
+  },
+)
+
+const state: TabsPanelApi = useTabsStateApi()
+const { active } = toRefs(state)
+const { selectTab, background } = state
+const tabIsActive = computed(() => props.name === active.value)
+
+const activateTab = () => {
+  selectTab(props.name)
+}
+
+watch(
+  () => props.disabled,
+  (newVal) => {
+    if (newVal && tabIsActive.value) {
+      selectTab('')
+    }
+  },
+)
+</script>
+
+<template>
+  <button
+    role="tab"
+    class="s-tab flex justify-center items-center sora-tpg-p2"
+    :disabled="disabled"
+    :class="[{ 's-tab_active': tabIsActive }, `s-tab_background-${background}`]"
+    @click="activateTab"

```

Comment:
About `s-tab_background-xxx` class. `background` here is a modifier name, and `xxx` is its value. So, the correct form would be `s-tab_background_xxx`.